homeassistant/components/planifneige/sensor.py

```diff
@@ -0,0 +1,132 @@
+"""
+Sensor for the City of Montreal's Planif-Neige snow removal APIs.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.planifneige/
+"""
+
+import logging
+
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.restore_state import RestoreEntity
+
+DEPENDENCIES = ['planifneige']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_STREETID = 'streetid'
+CONF_STREETS = 'streets'
+CONF_DBPATH = 'database_path'
+
+ATTR_SENSOR = 'sensor'
+ATTR_STATES = 'states'
+
+DOMAIN = 'planifneige'
+
+STREET_STATE = {
+    '0': ['Snowed', 'mdi:snowflake'],  # street is snowed in
+    '1': ['Clear', 'mdi:road'],  # street is clear
```

Comment:
Will this be the default if there is no snow?

Reply:
Yes, state = clear (1) is the default if there is no snow.